sample/src/main/java/com/zhihu/matisse/sample/CustomMatisseActivity.java

```diff
@@ -143,6 +147,9 @@ public void onClick(View v) {
                 .theme(theme)
                 .delegate(this)
                 .allowsMultipleSelection(true)
+                .maxVideoLength(120)
```

Comment:
I know I pass a global variable from JS going into image picker and pass to `Matisse`, please consider that.

Reply:
Same with the one above, this is only for the sample app that is included in matisse

Reply:
Got it